server/network.js

```diff
@@ -347,89 +331,90 @@ network.readNetwork_ = function(networkFile) {
   var isFirst = true;
   var valueNames = [];
   var lines = fs.readFileSync(networkFile).toString().split('\n');
-  for (var lineNum in lines) {
-    var line = lines[lineNum];
-    if (!validFile) continue;
+  lines.forEach(function(line) {
+    if (!validFile) return;
     var parts = line.split(/[\t\s]+/);
+    var source = parts[0];
+    var target = parts[1];
     if (parts.length < 3) {
       validFile = false;
-      continue;
+      return;
     }
     if (isFirst) {
       isFirst = false;
       for (var i = 2; i < parts.length; i++) {
         valueNames.push(parts[i]);
       }
-      continue;
+      return;
```

Comment:
I think in forEach(function()), return means continue.. Or I should use continue?